registry/assorted_layout_widgets.test

```diff
@@ -0,0 +1,7 @@
+contact=marcglasberg@gmail.com
+contact=pascal@pascalwelsch.com
+
+fetch=git clone https://github.com/marcglasberg/assorted_layout_widgets.git tests
+fetch=git -C tests checkout 388db9ea38df1e0991ffdbe8e790699fb089202a
+update=.
+test=flutter test
```

Comment:
nit: newline at EOF

Reply:
@marcglasberg or @passsy - would this actually have failed at times where things broke? Looking at tests in this repo, I don't see any that actually exercise widget logic.

Maybe there would have been compilation errors, but it's not clear to me.

Reply:
Both would have been compilation errors